components/json-fetch/json-fetch.js

```diff
@@ -136,6 +136,13 @@ $document.on( fetchEvent, function( event ) {
 					}
 				}
 
+				// Ensure we only receive JSON data and don't allow jsonp
+				// jQuery will raise an error if other data format is received
+				fetchOpts.dataType = "json";
+				if ( fetchOpts.jsonp ) {
+					fetchOpts.jsonp = false;
+				}
+
```

Comment:
Should the chat wizard have the same conditional [before line 94](https://github.com/wet-boew/GCWeb/blob/master/components/wb-chtwzrd/chatwizard.js#L94)?